web-common/src/features/dashboards/pivot/pivot-column-definition.ts

```diff
@@ -180,8 +176,8 @@ export function getColumnDefForPivot(
 
     return {
       label: measure?.label || m,
-      formatter: createMeasureValueFormatter<null | undefined>(measure),
```

Comment:
All table measure formatting across the application uses `createMeasureValueFormatter`. Would be better to stick with it.